versioned_docs/version-3.x/developer/order-expiration.mdx

```diff
@@ -0,0 +1,87 @@
+---
+title: Order Expiration
+---
+
+:::info
+
+expireOrdersAfter was introduced in **Saleor 3.13**
+
+deleteExpiredOrdersAfter was introduced in **Saleor 3.14**
+
+:::
+
+:::caution
+
+This feature is in the **Feature Preview** stage, which means that it is available for experimentation and
+feedback. However, it is still undergoing development and is subject to modifications.
+
+:::
+
+## Introduction
+
+Saleor allows finalizing checkout without payment. This can be achieved by setting `orderSettings.allowUnpaidOrders` for a given `Channel` to `true` and calling `checkoutComplete` without any payment/transaction object or by using the `orderCreateFromCheckout` mutation with first order then payment flow.
+
+If the payment process is canceled by the customer and the order remains unpaid for some time, it is possible to set automatic stock releases and remove the order.
+
+## Configuration
+
+To set the expiration of orders, specify the number of minutes after which the orders should expire by setting `expireOrdersAfter`.
+Additionally, the `deleteExpiredOrdersAfter` value can be modified. By default, this is set to 60 days, but it can be changed to anywhere between 1 and 120 days.
+
+```graphql
+mutation ChannelUpdate {
+  channelUpdate(
+    id: "Q2hhbm5lbDoyMjQz"
+    input: {
+      orderSettings: { expireOrdersAfter: 360, deleteExpiredOrdersAfter: 30 }
+    }
+  ) {
+    channel {
+      id
+      orderSettings {
+        expireOrdersAfter
+        deleteExpiredOrdersAfter
+      }
+    }
+    errors {
+      field
+      code
+      message
+    }
+  }
+}
+```
+
+```json
+{
+  "data": {
+    "channelUpdate": {
+      "channel": {
+        "id": "Q2hhbm5lbDoyMjQz",
+        "orderSettings": {
+          "expireOrdersAfter": 360,
+          "deleteExpiredOrdersAfter": 30
+        }
+      },
+      "errors": []
+    }
+  },
+  "extensions": {
+    "cost": {
+      "requestedQueryCost": 0,
+      "maximumAvailable": 50000
+    }
+  }
+}
+```
+
+## How it works
+
+Once an unpaid order is created (via [`checkoutComplete`](../../../api-reference/checkout/mutations/checkout-complete) or [`orderCreateFromCheckout`](api-reference/orders/mutations/order-create-from-checkout.mdx)), it remains in `UNCONFIRMED` status. If the customer does not complete the payment within `expireOrdersAfter` minutes after that, the status will change to `EXPIRED` and stock allocation will be released.
+After `deleteExpiredOrdersAfter` days orders in `EXPIRED` status will be deleted and disappear from the order list.
+
+## How to disable order expiration
+
+To disable order expiration, set `expireOrdersAfter` to 0.
+
+When `expireOrdersAfter` is set to 0, the orders will not expire. Since there will be no orders with status `EXPIRED` Saleor will not automatically delete such orders.
```

Comment:
```suggestion
When `expireOrdersAfter` is set to 0, the orders will not expire. Since there will be no orders with the status `EXPIRED` Saleor will not automatically delete such orders.
```